english/vote-the-president/trump/trump.md

```diff
@@ -8,7 +8,7 @@ Know about [Latest hit from Trump Floyd "The Wall"](./trump-floyd-the-wall/thewa
 
 You must have a tiny.. pair of hands.
 
+Ashamed at your poor electoral choice, and fearful of the consequences of Trump's Presidency, you decide to flee the country.  
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.